src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -168,3 +173,32 @@ export class Feedback implements BaseEntity {
         }
     }
 }
+
+/**
+ * Helper method to build the feedback text for the review. When the feedback has a link with grading instruction
+ * it merges the feedback of the grading instruction with the feedback text provided by the assessor. Otherwise,
+ * it return detail_text or text property of the feedback depending on the submission element.
+ *
+ * @param feedback that contains feedback text and grading instruction
+ * @returns {string} formatted string representing the feedback text ready to display
+ */
+export const buildFeedbackTextForReview = (feedback: Feedback): string => {
```

Comment:
My comment regarding having this method as `static`, was not about removing the `static` keyword, but rather making it a method of the `Feedback` class, meaning:
```ts
class Feedback {
    // some declarations
    ...

    buildFeedbackTextForReview(): string { 
        // code here
    }
}
```

Unless there is a strong reason not to make it so.

Do we also want to update the existing `Feedback` object?
Would it make sense to do it upon getting the `Feedback` object from the server (on construction of the Feedback) instead of calling it in every component?

Reply:
ohh I see what you meant now. When I remove only static keyword, I could not use\invoke the method. It did not recognize the method, maybe it is due to lack of my knowledge. 

>Do we also want to update the existing Feedback object?
no, building the text is only for showing the combined feedback in UI when the students review their submission. We do not store these two feedback (comes from grading instruction and detail_text or text of feedback object) in one field, so I do not think moving this logic into backend side. What do you think?